src/components/MegasetCard/index.tsx

```diff
@@ -27,6 +27,8 @@ const MegasetCard: React.FunctionComponent<MegasetCardProps> = ({
     // container instead of the container expanding to fit publications
     const maxWidth = numDatasets > 1 ? "100%" : datasetCardWidth;
 
+    // basic assumption here is that shorter titles may indicate primacy 
+    const datasets = reverse(sortBy(map(megaset.datasets), "title"))
```

Comment:
the comment says shorter titles but is this an alphabetical order string sort?

Reply:
Why not just add some kind of metadata array that defines the order so that the title can be anything?

Reply:
ok yeah, we can add that on the data side.... my understanding of this issue based on the slack comments was that the issue was the randomness more than the actual order so this seemed like a good solution to fix that problem

Reply:
Regarding the alphabetical sort: it was reverse alphabetical. I just meant that if you have `dataset` and `dataset_another_thing`, we'd want to have the dataset first. I just made a change that makes it clearer